pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: seeds
 description: SEEDS Wallet
 
-version: 1.0.1+12
+version: 1.1.0+4
```

Comment:
Why 1.1 and why +4?

Reply:
I don't know flutter did that too. And now we have 1.1 we can't go back easily either.

Reply:
I happened to roll up to 1.1 some time ago before I uploaded a version to testflight so that got the iOS version stuck on >= 1.1. The +4 is just the build number. I very much doubt that any user cares if the app version is 1.0 or 3.45... I've never met anyone other than developers who even mention version numbers :D